sleepecg/io/sleep_readers.py

```diff
@@ -0,0 +1,269 @@
+# Authors: Florian Hofer
+#
+# License: BSD (3-clause)
+
+"""Read datasets containing ECG data and sleep stage annotations."""
+
+import datetime
+from dataclasses import dataclass
+from enum import IntEnum
+from pathlib import Path
+from typing import Iterator, NamedTuple, Optional, Union
+from xml.etree import ElementTree
+
+import numpy as np
+
+from ..config import get_config
+from ..heartbeats import detect_heartbeats
+from .nsrr import get_nsrr_url, list_nsrr
+from .utils import download_file
+
+__all__ = [
+    'SleepRecord',
+    'SleepStage',
+    'read_mesa',
+]
+
+
+class SleepStage(IntEnum):
+    """
+    Mapping of AASM sleep stages to integers.
+
+    To facilitate hypnogram plotting, values increase with wakefulness.
```

Comment:
```suggestion
    This facilitates hypnogram plotting (values increase with wakefulness).
```

Reply:
I'd leave this as it is, since it should explain why the values increase with wakefulness, not why the enum is there in the first place (which is to avoid having "magic" numeric literals in the reader functions). Or reformulate to make the intention clearer, e.g.
```suggestion
    The values increase with wakefulness, so the order of stages will be
    correct in a hypnogram plot.
```

Reply:
Ah, sorry, I misread that. Then keep the original please.